addons/wiki/models.py

```diff
@@ -200,6 +301,15 @@ def save(self, *args, **kwargs):
             self.node.update_search()
         return rv
 
+    def update(self, user, content):
```

Comment:
Can we just refactor these into the same function?

Reply:
yeah good idea, no need for both